source/MaterialXGenOsl/OslShaderGenerator.cpp

```diff
@@ -10,6 +10,7 @@
 namespace MaterialX
 {
 const string OslShaderGenerator::LANGUAGE = "genosl";
+const string OslShaderGenerator::TARGET = "general";
```

Comment:
What is a good target name to use here?

Reply:
Maybe "vanilla", to indicate this is pure OSL without any renderer extensions.